README.md

```diff
@@ -9,6 +9,7 @@
 | cpu-unit-test | [![Build Status](https://dev.azure.com/msrasrg/SuperBenchmark/_apis/build/status/microsoft.superbenchmark?branchName=main)](https://dev.azure.com/msrasrg/SuperBenchmark/_build/latest?definitionId=77&branchName=main) |
 | gpu-unit-test | [![Build Status](https://dev.azure.com/msrasrg/SuperBenchmark/_apis/build/status/cuda-unit-test?branchName=main)](https://dev.azure.com/msrasrg/SuperBenchmark/_build/latest?definitionId=80&branchName=main) |
 
+
 __SuperBench__ is a validation and profiling tool for AI infrastructure.
```

Comment:
infrastructure => infrastructures

Reply:
infrastructure is always used as a non-countable noun. Please check the [Ref_1](https://english.stackexchange.com/questions/287855/is-it-correct-to-use-infrastructure-as-a-countable-noun-in-this-context) or [Ref_2](https://forum.wordreference.com/threads/infrastructure-or-infrastructures.3615565/)

Reply:
I see.